app/org/maproulette/controllers/api/TaskController.scala

```diff
@@ -221,6 +225,151 @@ class TaskController @Inject()(override val sessionManager: SessionManager,
     super.inject(obj)
   }
 
+  /**
+    * Gets and claims a task that needs to be reviewed.
+    *
+    * @param id Task id to work on
+    * @return
+    */
+  def startTaskReview(id:Long) : Action[AnyContent] = Action.async { implicit request =>
+    this.sessionManager.authenticatedRequest { implicit user =>
+      val task = this.dal.retrieveById(id) match {
+        case Some(t) => t
+        case None => throw new NotFoundException(s"Task with $id not found, cannot start review.")
+      }
+
+      val result = this.dal.startTaskReview(user, task)
+      Ok(Json.toJson(result))
+    }
+  }
+
+  /**
+    * Releases a claim on a task that needs to be reviewed.
+    *
+    * @param id Task id to work on
+    * @return
+    */
+  def cancelTaskReview(id:Long) : Action[AnyContent] = Action.async { implicit request =>
+    this.sessionManager.authenticatedRequest { implicit user =>
+      val task = this.dal.retrieveById(id) match {
+        case Some(t) => t
+        case None => throw new NotFoundException(s"Task with $id not found, cannot cancel review.")
+      }
+
+      val result = this.dal.cancelTaskReview(user, task)
+      Ok(Json.toJson(result))
+    }
+  }
+
+  /**
+    * Gets and claims the next task that needs to be reviewed.
+    *
+    * @return
+    */
+  def nextTaskReview(sort:String, order:String) : Action[AnyContent] = Action.async { implicit request =>
+    this.sessionManager.authenticatedRequest { implicit user =>
+      SearchParameters.withSearch { implicit params =>
+        //cs => "my challenge name"
+        //o => "mapper's name"
+        //r => "reviewer's name"
+        println("NEXT TASK REVIEW. Params?")
```

Comment:
These look like lines you would insert to debug something. Probably shouldn't check this in.